backend/src/test/java/gov/cdc/usds/simplereport/api/graphql/OrganizationFacilityTest.java

```diff
@@ -32,6 +36,41 @@ void createOrganization_siteAdminUser_ok() {
         orgCreated.path("createOrganization").path("name").asText());
   }
 
+  @Test
+  void setOrganizationIdentityVerified_orgUser_failure() {
+    ObjectNode variables =
+        JsonNodeFactory.instance
+            .objectNode()
+            .put("externalId", "THIS_DOES_NOT_MATTER")
+            .put("verified", false);
+    runQuery("set-organization-identity-verified", variables, ACCESS_ERROR);
+  }
+
+  @Test
+  void setOrganizationIdentityVerified_siteAdminUser_ok() {
+    TestUserIdentities.withUser(
+        TestUserIdentities.SITE_ADMIN_USER,
+        () -> {
+          Organization org = _dataFactory.createValidOrg();
```

Comment:
Only this line actually needs to be in the `withUser` block, no?

Reply:
that's what i thought initially, but then i have to reference `org` later in the code block, so i think it's all or nothing

Reply:
unless there's some secret Java way around this